.github/workflows/commit-message-check.yaml

```diff
@@ -48,7 +48,25 @@ jobs:
       uses: tim-actions/commit-message-checker-with-regex@v0.3.1
       with:
         commits: ${{ steps.get-pr-commits.outputs.commits }}
-        pattern: '^.+(\n.{0,72})*$|^.+\n\s*[^a-zA-Z\s\n]|^.+\n\S+$'
+        # Notes:
+        #
+        # - The subject line is not enforced here (see other check), but has
+        #   to be specified at the start of the regex as the action is passed
+        #   the entire commit message.
+        #
+        # - Body lines *can* be longer than the maximum if they start
+        #   with a non-alphabetic character.
+        #
+        #   This allows stack traces, log files snippets, emails, long URLs,
+        #   etc to be specified. Some of these naturally "work" as they start
+        #   with numeric timestamps or addresses. Emails can but quoted using
+        #   the normal ">" character, markdown bullets ("-", "*") are also
+        #   useful for lists of URLs, but it is always possible to override
+        #   the check by simply space indenting the content you need to add.
+        #
+        # - A SoB comment can be any length (as it is unreasonable to penalise
+        #   people with long names/email addresses :)
+        pattern: '^.+(\n([a-zA-Z].{0,71}|[^a-zA-Z\n].*|Signed-off-by:.*|))+$'
```

Comment:
Why did you delete the single word line support with the last part of original regex `^.+\n\S+$`

Reply:
FYI: https://github.com/kata-containers/tests/blob/master/cmd/checkcommits/checkcommits.go#L201

Reply:
The other parts looks good to me

Reply:
What about this one: `^.+(\n\s*([a-zA-Z].{0,71}|[^a-zA-Z\s\n].*|Signed-off-by:.*|\S+\s*))+$`
1. Support single word line
2. Because there will be spaces in the beginning of the line, add `\s*` after the `\n`
3. long lines should start with non-alphabetic character but not a space so I add \s in the part `[^a-zA-Z\n]`

Reply:
> Why did you delete the single word line support with the last part of original regex

It's redundant fwics: `[a-zA-Z].{0,71}` matches any number of words due to the dot.

> long lines should start with non-alphabetic character but not a space

We've always allowed long lines if they start with a space (see https://github.com/kata-containers/tests/tree/master/cmd/checkcommits#handling-long-lines), but that should be the exception. "Normal" lines should *not* start with a space.

Reply:
1. The dot only match one character, so `[a-zA-Z].{0,71}` can only match a word with length less than 72. But I think we should match words of any length in order to support url.
2. The space problem is my misunderstanding.

So I think the regex could be:
`^.+(\n([a-zA-Z].{0,149}|[^a-zA-Z\n].*|Signed-off-by:.*|\S+\s*))+$`

Reply:
any updates about the single word line? @jodh-intel